test/helpers/getPerformanceTests.ts

```diff
@@ -64,13 +65,18 @@ const createNewRunnableTests = async (numberOfTest) => {
 };
 
 export default function getPerformanceTests() {
-	if (process.argv.includes('--regenerate')) {
-		const newNumber = Number(process.argv[process.argv.length - 1]);
-
+	const regenerateIndex = process.argv.indexOf('--regenerate');
+	if (regenerateIndex !== -1) {
+		const nextArg = process.argv[regenerateIndex + 1];
 		const numberOfTest =
-			typeof newNumber === 'number' && !isNaN(newNumber) && newNumber > 0
-				? newNumber
-				: DEFAULT_NUMBER_OF_TEST;
+			nextArg === undefined
+				? DEFAULT_NUMBER_OF_TEST
+				: // tslint:disable-next-line: radix
```

Comment:
Let's just give it the radix instead of disabling tslint for it i.e. `parseInt(nextArg, 10)`